shared/models/videos/video-blacklist.model.ts

```diff
@@ -3,3 +3,18 @@ export interface BlacklistedVideo {
   videoId: number
   createdAt: Date
 }
+
+export interface RestBlacklistedVideo {
```

Comment:
We put in `shared` objects that are shared with the client so the formated videos. Please only export one interface that include all fields sent by the server. 

If you need more fields or methods in the client you can create another model file like we do for the Video model for example: https://github.com/Green-Star/PeerTube/blob/a9c5666773a2ac0e3b06ed6b3e717519b9f561e3/client/src/app/videos/shared/video.model.ts